packages/router/src/grpc/view-protocol-server/unbonding-tokens-by-address-index/index.ts

```diff
@@ -0,0 +1,39 @@
+import {
+  BalancesRequest,
+  UnbondingTokensByAddressIndexRequest_Filter,
+  UnbondingTokensByAddressIndexResponse,
+} from '@buf/penumbra-zone_penumbra.bufbuild_es/penumbra/view/v1/view_pb';
+import { Impl } from '..';
+import { balances } from '../balances';
+import { getIsClaimable, isUnbondingTokenBalance } from './helpers';
+
+export const unbondingTokensByAddressIndex: Impl['unbondingTokensByAddressIndex'] =
+  async function* (req, ctx) {
+    for await (const balancesResponse of balances(
+      new BalancesRequest({ accountFilter: req.addressIndex }),
+      ctx,
+    )) {
+      if (!isUnbondingTokenBalance(balancesResponse)) continue;
+      const claimable = await getIsClaimable(balancesResponse, ctx);
+
+      // See https://github.com/typescript-eslint/typescript-eslint/issues/7114
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-enum-comparison
```

Comment:
I wonder if we should disable this rule globally with this comment until the bug is fixed